spec/checkers/foreign_key_type_checker/mysql_spec.rb

```diff
@@ -32,13 +32,13 @@
     context 'when base key has integer type' do
       let(:base_type) { :integer }
 
-      include_examples 'check matches', [], %i[serial]
+      include_examples 'check matches', %i[integer], %i[bigint]
```

Comment:
MySQL specs used `serial` type but I didn't find anything about it in docs, so this looks like a typo to me. Let me know if I understood something wrong.